Add Indonesian translation

imanudinsh opened the pull request:
### Changes
Add Indonesian translation

### Testing
- [x] Tested on a physical device
- [ ] Added or modified unit test cases

### Others
- Fixes #0
- Connects #200

ashutoshgngwr commented on fastlane/metadata/android/in-ID/short_description.txt:
```diff
@@ -0,0 +1 @@
+Santai, tingkatkan fokus, dan tingkatkan produktivitas dengan derau pelataran minimal.
```
The short description needs to be 80 characters at most. It's 86 right now. Can you please rephrase it to be <= 80?

ashutoshgngwr commented on app/src/main/res/values-in-rID/strings.xml:
```diff
@@ -0,0 +1,168 @@
+  <string name="preset_already_exists">Preset dengan nama yang diberikan sudah ada!</string>
+  <string name="preset_saved">Preset disimpan!</string>
+  <string name="saved_presets">Preset Tersimpan</string>
+  <string name="preset_delete_confirmation">Konfirmasi hapus% s?</string>
```
There's a typo here.

```suggestion
  <string name="preset_delete_confirmation">Konfirmasi hapus %s?</string>
```

ashutoshgngwr commented on app/src/main/res/values-in-rID/strings.xml:
```diff
@@ -0,0 +1,168 @@
+    pengatur waktu.
+</string>
+  <string name="reset">Setel ulang</string>
+  <string name="auto_sleep_schedule_cancelled">Pengatur waktu tidur otomatis dibatalkan!</string>
```
Google translates it to _Sleep timer automatically canceled!_ but it needs to imply _Sleep timer cancelled!_ or _Automatic sleep timer cancelled!_

ashutoshgngwr reviewed (changes requested):
Hi @imanudinsh 

Thank you for taking the time to translate Noice to Indonesian. I have requested a few changes. Please have a look. In addition to those changes, if you could also remove the region codes from all file paths. On Android, it will work but is not required since Indonesian is only spoken in Indonesia. In Fastlane metadata, it will not work since Google Play only recognises `id` tag. So to sum it up,

- Rename `values-in-rID` to `in`
- Rename `fastlane/metadata/android/in-ID` to `fastlane/metadata/android/id`

imanudinsh commented:
Hello @ashutoshgngwr,

Thanks for reviewing my pull request, I'll fix it and come back asap

imanudinsh commented:
Hi @ashutoshgngwr,

I've fixed typos, updated translations, and changed the region code on all file paths.
please let me know if it still needs improvement.

ashutoshgngwr reviewed (approved):
Thanks, @imanudinsh! This is great work!
